File: SDKs/amazonaws.com/application-autoscaling/2016-02-06/python/src/sdk/models/shared/adjustmenttype_enum.py
from dataclasses import dataclass, field
from enum import Enum

class AdjustmentTypeEnum(str, Enum):
    CHANGE_IN_CAPACITY = "ChangeInCapacity"
    PERCENT_CHANGE_IN_CAPACITY = "PercentChangeInCapacity"
    EXACT_CAPACITY = "ExactCapacity"

File: SDKs/amazonaws.com/elasticmapreduce/2009-03-31/go/pkg/models/shared/adjustmenttypeenum.go
package shared

type AdjustmentTypeEnum string

const (
	AdjustmentTypeEnumChangeInCapacity        AdjustmentTypeEnum = "CHANGE_IN_CAPACITY"
	AdjustmentTypeEnumPercentChangeInCapacity AdjustmentTypeEnum = "PERCENT_CHANGE_IN_CAPACITY"
	AdjustmentTypeEnumExactCapacity           AdjustmentTypeEnum = "EXACT_CAPACITY"
)
